pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -171,9 +171,12 @@ public ServerType getServerType() {
       public static final String SQL = "sql";
       public static final String TRACE = "trace";
       public static final String DEBUG_OPTIONS = "debugOptions";
+      public static final String QUERY_OPTIONS = "queryOptions";
```

Comment:
I thought we had this to support partition @mayankshriv ?

Reply:
I don't recall using it for partitioning. But seems this got introduced in: https://github.com/apache/incubator-pinot/pull/1490